### PR TITLE
Fix #4207: json schema files for connections generated during build time

### LINKED ISSUE #4207: UI: json schema files for connections should be generated during build time
We are generating JSON schema files for connections and checking a copy of these forms to the source code repo. Instead, we should generate this ref resolved copy of JSON schemas during build time for the Typescript to consume it.

### PR DESCRIPTION
**Pull request by darth-coder00**
Closes #4207
### Describe your changes :

I worked on the connection parsing script to dereference schemas pre-build

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui @harshach

### CODE REVIEW

**Comment by vivekratnavel** [comment]
@darth-coder00 Great! Can we also delete all the json schema filesthat are checked in? In a later patch we can add it to gitignore.

**Comment by darth-coder00** [comment]
> @darth-coder00 Great! Can we also delete all the json schema files that are checked in? In a later patch we can add it to gitignore.

@vivekratnavel Done

**Review by vivekratnavel — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/parseConnectionSchema.js [hunk 1]
```diff
@@ -45,15 +53,37 @@ function traverseDirectory(Directory) {
   });
 }
 
+function copySourceFiles() {
+  try {
+    fse.copySync(schemaDir, `${rootDir}/schema`);
```
[the remark this answers is not in the document]
Comment (darth-coder00) [reply openmetadata-ui/src/main/resources/ui/parseConnectionSchema.js:58]: It's taken care of in the script `parse-conn-schema` as it was breaking the logic here

**Comment by vivekratnavel** [comment]
CI fails with an error:
```
[INFO] --- frontend-maven-plugin:1.11.0:yarn (yarn run build-check) @ openmetadata-ui ---
[INFO] Running 'yarn run build-check' in /home/runner/work/OpenMetadata/OpenMetadata/openmetadata-ui/src/main/resources/ui
[INFO] yarn run v1.22.18
[INFO] $ yarn run lint && yarn run parse-conn-schema
[INFO] $ eslint "./src/**/*.{js,jsx,ts,tsx}"
[INFO] $ node parseConnectionSchema && rm -rf connTemp && yarn run pretty-conn-schema
[INFO] Error: ENOENT: no such file or directory, stat 'src/jsons/connectionSchemas/connections'
[INFO]     at Object.statSync (node:fs:1538:3)
[INFO]     at __node_internal_ (node:internal/fs/utils:793:8)
[INFO]     at Object.rmdirSync (node:fs:1156:15)
[INFO]     at main (/home/runner/work/OpenMetadata/OpenMetadata/openmetadata-ui/src/main/resources/ui/parseConnectionSchema.js:67:8)
[INFO]     at Object.<anonymous> (/home/runner/work/OpenMetadata/OpenMetadata/openmetadata-ui/src/main/resources/ui/parseConnectionSchema.js:89:1)
[INFO]     at Module._compile (node:internal/modules/cjs/loader:1103:14)
[INFO]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
[INFO]     at Module.load (node:internal/modules/cjs/loader:981:32)
[INFO]     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
[INFO]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
[INFO]   errno: -2,
[INFO]   syscall: 'stat',
[INFO]   code: 'ENOENT',
[INFO]   path: 'src/jsons/connectionSchemas/connections'
```

**Review by vivekratnavel — APPROVED** [review]

**Pull request merged**